### ISSUE TITLE
Base64DataUrl ngf-pattern validation error

### ISSUE THREAD

**Issue by tiberiuzuld** [danialfarid/ng-file-upload#1294]
Hello,

Have an issue with validation on a Base64Data image.
If i put on the model a base64 image I get this error:
```javascript
TypeError: files.splice is not a function
    at validateSync (ng-file-upload.js:1231)
    at upload.validate (ng-file-upload.js:1247)
```

I get this error when I drag or select a new image and I convert the image to base64 which is the format I need to send to the backend.

Created an example with the issue [Plunker](https://plnkr.co/edit/OE97515YOJtvSl7sMNFp?p=preview)

Let me know if you need more info.

**Comment by danialfarid** [comment]
Should  be fixed at 11.2.2

**Comment by tiberiuzuld** [comment]
Thank you :D
